### PR TITLE
chore(parse5): Update logic to work w/ parse5@2.*

### PR DESCRIPTION
**Pull request by MarkPieszak**

### CODE REVIEW

**Comment by gdi2290** [comment]
@MarkPieszak what about the Parse5Adapter?  https://github.com/angular/angular/blob/2953ea10a7638d05b64486cdec5c6a240a8d675e/modules/%40angular/platform-server/src/parse5_adapter.ts

File: modules/universal/src/node/platform/dom/node_dom_renderer.ts [hunk 1]
```diff
@@ -249,6 +251,16 @@ export class NodeDomRenderer extends DomRenderer {
     }
     return super.invokeElementMethod(location, methodName, args);
   }
+  
+  _setCheckedAttribute(renderElement, propertyName, propertyValue) {
```
Comment (gdi2290) [inline modules/universal/src/node/platform/dom/node_dom_renderer.ts]: can you make a separate PR for this so we can merge it sooner

Comment (MarkPieszak) [inline modules/universal/src/node/platform/dom/node_dom_renderer.ts]: Will do
